assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image-next-previous/settings.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import {
+	PanelBody,
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
```

Comment:
Same here:

```suggestion
	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
	// @ts-ignore - Ignoring because `__experimentalToggleGroupControlOption` is not yet in the type definitions.
	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
```